### ISSUE TITLE
Add override keyword

### ISSUE THREAD

**Issue by adrian-tabirta** [Scandit/scandit-react-native-datacapture-core#4]
Please update next files: 

- ScanditDataCaptureBarcodeCapture
- ScanditDataCaptureBarcodeTracking

method:
```
  @objc func invalidate()  -> @objc override func invalidate()  
```

**Comment by ScanditSupport** [comment]
@adrian-tabirta thanks for your comment. We'll look into it and provide an update.

**Comment by ScanditSupport** [comment]
@adrian-tabirta the issue seems depending on the react native version. However, we'll provide a fix in our next release v6.8 which is supposed to be released next month. 
If you want us to send a reminder once it's released, please reach out to us via support.scandit.com
